numpy/core/src/common/npy_cpu_features.c

```diff
@@ -22,6 +22,15 @@ npy__cpu_init_features(void);
 static int
 npy__cpu_try_disable_env(void);
 
+/*
+ * Enable CPU dispatched features at runtime if environment variable
+ * 'NPY_ENABLE_CPU_FEATURES' is defined.
+ * Multiple features can be present, and separated by space, comma, or tab.
+ * Raises an error if parsing fails or if the feature was not enabled
+*/
+static int
+npy__cpu_try_enable_env(void);
+
```

Comment:
```suggestion
/*
 * Enable or disable CPU dispatched features at runtime if the environment variable `NPY_ENABLE_CPU_FEATURES`  or  `NPY_DISABLE_CPU_FEATURES` depends on the value of boolean parameter `disable`(toggle).
 *
 * Multiple features can be present, and separated by space, comma, or tab.
 * Raises an error if parsing fails or if the feature was not enabled or disabled
*/
npy__cpu_check_env(bool disable);
```
please keep it simple, handle both operations within one function and use boolean param to toggle between them.

Reply:
Sounds good, will rework when I get a chance 👍